test.sh

```diff
@@ -10,7 +10,7 @@ sqlite3 "$testdir/core.db" < sql/init_data/init_core.sql;
 sqlite3 "$testdir/common.db" < sql/init_data/init.sql;
 sqlite3 "$testdir/selectfunc.db" < sql/init_data/init_selectfunc.sql;
 
-sed -i 's/REGRESS =.*/REGRESS = extra\/sqlite_fdw_post extra\/bitstring extra\/bool extra\/float4 extra\/float8 extra\/int4 extra\/int8 extra\/numeric extra\/out_of_range extra\/timestamp extra\/uuid extra\/join extra\/limit extra\/aggregates extra\/prepare extra\/select_having extra\/select extra\/insert extra\/update extra\/encodings sqlite_fdw type aggregate selectfunc /' Makefile
+#sed -i 's/REGRESS =.*/REGRESS = extra\/sqlite_fdw_post types\/bitstring types\/bool types\/float4 types\/float8 types\/int4 types\/int8 types\/macaddr types\/macaddr8 types\/numeric types\/out_of_range types\/timestamp types\/uuid extra\/join extra\/limit extra\/aggregates extra\/prepare extra\/select_having extra\/select extra\/insert extra\/update extra\/encodings sqlite_fdw type aggregate selectfunc /' Makefile
```

Comment:
> Could you explain why you comment out this command?
Because this is default test sequence, which is well defined in `Makefile`.

> If you remove it, you always have to define a REGRESS variable before testing 

Yes. This is my proposed changes. But _before testing_ is not outside of `testN.sh` files.

@son-phamngoc , in my case for `test1.sh` and `test2.sh` I changed `sed` to `export REGRESS=....` inside of `testN.sh`, hence no `mdate` or content of `Makefile` are changed.

My example solution inside of some `tesN.sh`, named `testMAC.sh
```sh
export REGRESS='type types/macaddr types/macaddr8 ';
make clean $1;
make $1;
make check $1 | tee make_check.out;
export REGRESS=;
```
I have no problems with testing process and no modifications for `Makefile` and _no quoting_ for `REGRESS` content.

> Step 1 and 3 needs to be executed manually every time you execute a test file.

No. Does my solution not work in your environment? I think `export REGRESS=` is simpler and faster than `sed` command with file modification command.

> This sed is our rule to avoid above problem, and you can see that we use this style in all our products.

Does my solution not resolve this problems? My idea is just default `test.sh` have no `export REGRESS` inside of file, but other `testN.sh` files has.

> Therefore, please revert it.

If my solution is unwanted for pgspider team, I can revert this, but i think described solution is simplification and have not found any arguments against it. Have you found after my description?

Reply:
@mkgrgis Thank you for your explanation.  
I want to avoid manually export variable because we cannot control if exported variable is correct or not. However, if you are going to export REGRESS in each test script, like https://github.com/pgspider/sqlite_fdw/commit/eaade87827cafedf70227a2367120375d1d041ed, it is OK for us.  
But please do not just add a comment like that. Could you export the correct REGRESS variable there?  
I mean like this:  
```
export REGRESS= extra/sqlite_fdw_post types/bitstring types/bool types/float4 types/float8 types/int4 types/int8 types/numeric types/macaddr types/macaddr8 types/out_of_range types/timestamp types/uuid extra/join extra/limit extra/aggregates extra/prepare extra/select_having extra/select extra/insert extra/update extra/encodings sqlite_fdw type aggregate selectfunc
```

Reply:
> Could you export the correct REGRESS variable there?

No problem, but in this case (default `test.sh`) void `REGRESS` variable yet cause general testing sequence. Do I need export _doubled_ `REGRESS` content anyway? If yes, I'll change this.

Reply:
@mkgrgis 
Yes, please keep `REGRESS` at 2 places:

- Please uncomment the line in https://github.com/pgspider/sqlite_fdw/commit/177ffdfb84601d0b49ff4a016fce194389bd9da6 of `test.sh`.  

- Please keep the `REGRESS` in Makefile as current. It will be useful when user want to execute `make check` without using `test.sh`. `ifndef` will help to avoid re-definition.

Reply:
Done in https://github.com/pgspider/sqlite_fdw/pull/101/commits/e942100f70ec76575607e6a08615dc85cc50407f , @son-phamngoc !

Reply:
Thank you. I confirmed. Please help me to resolve this comment and https://github.com/pgspider/sqlite_fdw/pull/101#discussion_r1883126721